packages/core/admin/ee/server/services/audit-logs/process-event.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+module.exports = function processEvent(strapi, name, ...args) {
+  const defaultGetPayload = (...args) => args[0];
+  const getAuthPayload = (...args) => ({
+    user: strapi.service('admin::user').sanitizeUser(args[0].user),
```

Comment:
I see the sanitization is done on emit for the entity manager:
https://github.com/strapi/strapi/blob/972bdd72cc07f8ae378f472162cd0f4371aa3f26/packages/core/content-manager/server/services/entity-manager.js#L15  Should we be doing it here or when we emit events? If we do it here should we refactor the entity manager `wrapEmitEvent`?

Reply:
Very interesting, thanks for linking this. I think it's probably better to sanitize when you're emitting the event. You probably don't want sensitive data in your event hub at all, it's not just about audit logs.

So I'd vote for not changing the event manager, removing our process event sanitizing, and making sure we sanitize the data in other places where events are emitted